cypress/integration/Images/LeadImageTest.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Lead Image tests
+
+    Scenario Outline: As a user, I want to view the Lead image and Caption on pages with lead image
+        Given user is navigating to "<url>"
+        Then lead image is visible
+        And "<caption>" caption is correct
+
+        Examples:
+            | url                                           | caption                                                                                                                                                   |
+            | /research-areas-genomics                      | This Circos plot visualizes data from The Cancer Genome Atlas (TCGA) and allows scientists to explore the interrelationships among different data points. |
+            | /espanol/espanol-investigacion-areas-genomica | El diagrama de Circos visualiza datos del Atlas del Genoma de Cáncer (TCGA) y permite a los científicos                                                   |
+            | /espanol/efectos-secundarios                  | Los implantes se insertan debajo de la piel o del músculo del pecho                                                                                       |
+            | /about-cancer/treatment/side-effects          | Tell your doctor about side effects you are experiencing, so you get the care and treatment you need to manage these problems.                            |
+
+    Scenario Outline: As a user, I want to view the Lead image and Credit on pages with lead image
+        Given user is navigating to "<url>"
+        Then lead image is visible
+        And "<credit>" credit is correct
```

Comment:
same here 
And image credit is "\<credit\>"

Reply:
fixed this